example/README.md

```diff
@@ -0,0 +1,55 @@
+# Proxmox/K3s Example
+
+This is an example project for setting up your own K3s cluster at home.
+
+## Summary
+
+### VMs
+
+This will spin up:
+
+- 1 support vm with api loadbalancer and k3s database with 2 cores and 4Gb mem
+- 2 master nodes with 2 cores and 4Gb mem
+- 1 node pool with 2 worker nodes each having 2 cores and 4Gb mem
+
+### Networking
+
+- The support VM will be spun up on `192.168.0.200`
+- The master VMs will be spun up on `192.168.0.201` and `192.168.1.202`
```

Comment:
The two ip are not `192.168.0.201` and 192.168.**0**.202
Same line bellow.

Reply:
Are you sure this is true? From the terraform, the support node gets the first IP in the control plane CIDER and the masters get the next N depending on how many you ask for.

In this case, the control plane CIDER is `192.168.0.200/29` which is the range `192.168.0.200` to `192.168.0.207`.

The first IP is `192.168.0.200` which is given to the support node, and the next two (`.201` and `.202`) are given to the two support nodes.

What behaviour are you experiencing here?

Reply:
Ahhh sorry I missed that! Thanks for the catch!